roles/provision-mcs-server-apb/tasks/provision-service-server.yml

```diff
@@ -0,0 +1,114 @@
+- name: Mobile Client Service Server deployment
+  openshift_v1_deployment_config:
+    name: '{{ mobile_client_service_deploymentconfig_name }}'
+    namespace: "{{ namespace }}"
+    labels:
+      app: mcs-server
+      service: mcs-server
+      name: mcs-server
+      mobile: enabled
+    replicas: 1
+    selector:
+      app: mcs-server
+      service: mcs-server
+    spec_template_metadata_labels:
+      app: mcs-server
+      service: mcs-server
+    spec_template_spec_service_account_name: '{{ proxy_serviceaccount_name }}'
+    containers:
+    - name: mcs-server
+      image: '{{ mobile_client_service_image }}:{{ mobile_client_service_image_tag }}'
+      imagePullPolicy: Always
+      env:
+      - name: SERVER_PORT
+        value: "4000"
+      ports:
+      - name: mcs-server
+        protocol: TCP
+        container_port: 4000
+    - name: mcs-server-oauth-proxy
+      image: '{{ proxy_image }}:{{ proxy_image_tag }}'
+      imagePullPolicy: Always
+      ports:
+      - containerPort: '{{ mobile_client_service_proxy_port }}'
+        name: public
+      args:
+      - --provider=openshift
+      - --openshift-service-account={{ proxy_serviceaccount_name }}
+      - --upstream=http://localhost:{{ mobile_client_service_port }}
+      - --openshift-sar={"namespace":"{{ namespace }}","resource":"deploymentconfigs","name":"mcs-server","verb":"update"}
```

Comment:
we can leave the option like this for now, but we probably will update it later.